src/main/java/lotto/domain/Lotto.java

```diff
@@ -2,28 +2,33 @@
 
 import java.util.List;
 import java.util.Objects;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
 
 public class Lotto {
-    private Numbers numbers;
+    private List<Integer> numbers;
```

Comment:
```suggestion
    private final List<Integer> numbers;
```

가급적 부수효과가 적은 불변 형태 오브젝트를 사용해 보면 좋을것 같습니다. (다른곳도 동일합니다.)